### ISSUE TITLE
divide-by-zero in sv1.sv1_cuts

### ISSUE THREAD

**Issue by moustakas** [desihub/desitarget#495]
It's *possible* that `dchisq` can be all zeros (it is for the mocks!), so would be good to protect against this possibility:
```
[snip]
/Users/ioannis/repos/desihub/desitarget/py/desitarget/sv1/sv1_cuts.py:413: RuntimeWarning: divide by zero encountered in true_divide
  morph2 = (dchisq[..., 1] - dchisq[..., 0])/dchisq[..., 0] < 0.01
[snip]
```

**Comment by moustakas** [comment]
@geordie666 In `select_mock_targets` I bypassed this exception by initializing the delta-chi2 array with unity in the zeroth element, so there's no divide-by-zero.  So you should feel free to close this issue if it's never encountered in the "real" target catalogs.

**Comment by geordie666** [comment]
Thanks, @moustakas. I don't think this *is* an issue for the real data, but I want to try to go through the code today and clean up divide-by-zero issues in general. I'll close this when I've done that.

**Comment by geordie666** [comment]
Addressed in #497.
